mkosi/__init__.py

```diff
@@ -1616,12 +1616,13 @@ def prepare_tree(args: CommandLineArguments, root: str, do_run_build_script: boo
         if args.output_format is OutputFormat.subvolume or (
             args.output_format is OutputFormat.gpt_btrfs and not args.minimize
         ):
-            btrfs_subvol_create(os.path.join(root, "home"))
-            btrfs_subvol_create(os.path.join(root, "srv"))
-            btrfs_subvol_create(os.path.join(root, "var"))
-            btrfs_subvol_create(os.path.join(root, "var/tmp"), 0o1777)
-            os.mkdir(os.path.join(root, "var/lib"))
-            btrfs_subvol_create(os.path.join(root, "var/lib/machines"), 0o700)
+            if not args.usr_only:
```

Comment:
maybe this also should check for is_generated_root()?

Reply:
maybe? i changed it to ```is_generated_root()``` include all cases